### ISSUE TITLE
Support custom BYO domains for Tiddlyhost sites

### ISSUE THREAD

**Issue by simonbaird** [simonbaird/tiddlyhost#145]
Something like https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site would be nice.

(Inspired by STobot and @pmario's thread on the TiddlyWiki mailing list.)

**Comment by pmario** [comment]
Hi, I personally would consider this a "premium" feature for payed accounts. ... There needs to be a mechanism in place, that allows you to handle a "letsencrypt" certificate handling, for BYO-domains.

**Comment by simonbaird** [comment]
Yeah, there would be some extra work needed to handle the cert recreation.

I think this isn't a priority in the short term, but I'd like to keep it in the backlog for some time in the future.

**Comment by simonbaird** [comment]
There's two levels:
* mydomain.com serves a single site.
* mydomain.com is configured with wildcard dns and is able to serve many sites, e.g. info.mydomain.com, coolstuff.mydomain.com, etc.

I guess the first one is a little easier than the second.

**Comment by flygohr** [comment]
Bump. Would love to pay for such feature
